packages/gatsby-plugin-sharp/src/gatsby-node.js

```diff
@@ -63,9 +63,52 @@ const finishProgressBar = () => {
 exports.onPostBuild = () => finishProgressBar()
 exports.onCreateDevServer = () => finishProgressBar()
 
-exports.onPreBootstrap = ({ actions }, pluginOptions) => {
+// TO-DO - figure this out before merging (it can't unconditionally return false),
+// so we don't end up with duplicate progress bars (one from this plugin and one from core)
+const GatsbyCoreVersionManagesProgressBar = () => false
```

Comment:
This is super important to figure out

Reply:
this is no longer in PR - but still is valid question

right now I tentatively use availability of (not yet implemented) `actions.setJobTypeMetadata` action